src/PKSim.Core/Services/DistributedParameterToTableParameterConverter.cs

```diff
@@ -275,20 +290,61 @@ private bool parameterShouldBeDefinedAsTable(IDistributedParameter parameter)
+      private void createPopulationTableParameter(
+         string individualParameterPath,
+         IDistributedParameter individualParameter,
+         PopulationSimulation populationSimulation,
+         IReadOnlyList<ParameterDistributionMetaData> allDistributionsForMaleParameter,
+         IReadOnlyList<ParameterDistributionMetaData> allDistributionsForFemaleParameter)
       {
-         if (populationSimulation == null) return;
-         var parameter = individualParameter.Value;
+         if (populationSimulation == null)
+            return;
 
-         addAgingDataToPopulationSimulation(populationSimulation, individualParameter.Key, parameter,
+         addAgingDataToPopulationSimulation(populationSimulation, individualParameterPath, individualParameter,
             p =>
             {
-               var distributions = allDistributionsWithAgeStrictBiggerThanOriginData(allDistributionsForMaleParameter, allDistributionsForFemaleParameter, p.OriginData).ToList();
+               var distributions = allDistributionsWithAgeStrictBiggerThanOriginData(allDistributionsForMaleParameter, allDistributionsForFemaleParameter, p.OriginData);
                return createTableFormulaFrom(p, distributions);
             });
       }
 
+      private void createPopulationHeightTableParameter(
+         string heightParameterPath,
+         IDistributedParameter meanHeightParameter,
+         PopulationSimulation populationSimulation
+      )
+      {
+         if (populationSimulation == null || !_createAgingDataInSimulation)
+            return;
+
+         var originData = _baseOriginData.Clone();
+         var allAges = populationSimulation.AllOrganismValuesFor(CoreConstants.Parameters.AGE, _entityPathResolver);
+         var allGAs = populationSimulation.AllOrganismValuesFor(Constants.Parameters.GESTATIONAL_AGE, _entityPathResolver);
+         var allHeights = populationSimulation.AllOrganismValuesFor(CoreConstants.Parameters.HEIGHT, _entityPathResolver);
+         var allGender = populationSimulation.AllGenders(_genderRepository).ToList();
+
+         var tableFormulaParameter = new TableFormulaParameter<IDistributedParameter> {OriginData = originData, Parameter = meanHeightParameter};
+         for (int individualIndex = 0; individualIndex < populationSimulation.NumberOfItems; individualIndex++)
+         {
+            //create origin data for individual i
+            originData.Age = allAges[individualIndex];
+            originData.GestationalAge = allGAs[individualIndex];
+            originData.Height = allHeights[individualIndex];
+            originData.Gender = allGender[individualIndex];
+            tableFormulaParameter.Value = originData.Height.Value;
+
+            var heightDistributions = heightDistributionsFor(originData);
+            tableFormulaParameter.Percentile = heightDistributions.currentPercentile;
```

Comment:
REquired to get the percentile of the current HEIGHT so that we can ensure perfect distributions scaling